Merge dev to Main

janblauer opened the pull request.

janblauer committed:
Idea. entfernt & spec datei gelöscht

janblauer committed:
Idea. entfernt & spec datei gelöscht

janblauer committed:
Erste Buttons hinzugefügt

janblauer committed:
Delete favicon icon

janblauer committed:
Add gitignore

janblauer committed:
remove idea files

janblauer committed:
Merge pull request #2 from JoghurtCode/Fix/Gitignore

Fix/gitignore

janblauer committed:
add angualr/ CICD

janblauer committed:
Merge pull request #3 from JoghurtCode/feature/CI_CD-&-Angualr

add angualr/ CICD

janblauer committed:
fix Ci/CD

janblauer committed:
Merge pull request #4 from JoghurtCode/feature/CI_CD-&-Angualr

fix Ci/CD

janblauer committed:
fix Ci/CD 2

janblauer committed:
Merge pull request #5 from JoghurtCode/feature/CI_CD-&-Angualr

fix Ci/CD 2

janblauer committed:
fix Ci/CD 2

janblauer committed:
Merge pull request #6 from JoghurtCode/feature/CI_CD-&-Angualr

fix Ci/CD 2

janblauer commented:
@JoghurtCode kannst du das beheben, brauche github pages

janblauer committed:
add dependabot.yml

janblauer committed:
Merge pull request #8 from JoghurtCode/feature/dependabot

add dependabot.yml

dependabot[bot] committed:
Bump typescript from 5.3.3 to 5.4.5 in /frontend

Bumps [typescript](https://github.com/Microsoft/TypeScript) from 5.3.3 to 5.4.5.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Changelog](https://github.com/microsoft/TypeScript/blob/main/azure-pipelines.release.yml)
- [Commits](https://github.com/Microsoft/TypeScript/compare/v5.3.3...v5.4.5)

---
updated-dependencies:
- dependency-name: typescript
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump zone.js from 0.14.5 to 0.14.6 in /frontend

Bumps [zone.js](https://github.com/angular/angular/tree/HEAD/packages/zone.js) from 0.14.5 to 0.14.6.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/main/packages/zone.js/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/zone.js-0.14.6/packages/zone.js)

---
updated-dependencies:
- dependency-name: zone.js
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

janblauer committed:
Merge pull request #9 from JoghurtCode/dependabot/npm_and_yarn/frontend/typescript-5.4.5

Bump typescript from 5.3.3 to 5.4.5 in /frontend

janblauer committed:
Merge pull request #10 from JoghurtCode/dependabot/npm_and_yarn/frontend/zone.js-0.14.6

Bump zone.js from 0.14.5 to 0.14.6 in /frontend

janblauer committed:
Aufsetzen von Angular neu

janblauer committed:
Merge pull request #11 from JoghurtCode/feature/AngularChanges

Aufsetzen von Angular neu

janblauer committed:
Hinzufügen von Design-Prototyp für die Componenten

janblauer committed:
Merge pull request #12 from JoghurtCode/feature/AngularChanges

Hinzufügen von Design-Prototyp für die Componenten

janblauer committed:
Hinzufügen von Design-Prototyp History Component

janblauer committed:
Merge pull request #13 from JoghurtCode/feature/AngularChanges

Hinzufügen von Design-Prototyp History Component

dependabot[bot] committed:
Bump typescript from 5.3.3 to 5.4.5 in /frontend

Bumps [typescript](https://github.com/Microsoft/TypeScript) from 5.3.3 to 5.4.5.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Changelog](https://github.com/microsoft/TypeScript/blob/main/azure-pipelines.release.yml)
- [Commits](https://github.com/Microsoft/TypeScript/compare/v5.3.3...v5.4.5)

---
updated-dependencies:
- dependency-name: typescript
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

janblauer committed:
Merge pull request #14 from JoghurtCode/dependabot/npm_and_yarn/frontend/typescript-5.4.5

Bump typescript from 5.3.3 to 5.4.5 in /frontend

janblauer committed:
Angular github pages try

janblauer committed:
Merge pull request #15 from JoghurtCode/Yaml-fix

Angular github pages try

janblauer committed:
Angular github pages try 3

janblauer committed:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	.github/workflows/angular.yml

janblauer committed:
Angular github pages try 4

janblauer committed:
Angular github pages try 5 :(

JoghurtCode reviewed (changes requested):
Ich finde, dass wir erstmal die pipeline für das frontend zum laufen bekommen sollten, bevor wir das mergen.

nxlsls committed:
Hinzufügen von REST-Controllern und Konfiguration für H2-Datenbank

- REST-Controller für UserProfile, GroceryItem und Meal erstellt
- Entsprechende Repository-Interfaces für UserProfile, GroceryItem und Meal hinzugefügt
- H2-Datenbank und Anwendungseigenschaften konfiguriert
- DataLoader hinzugefügt, um die Datenbank mit Beispieldaten zu initialisieren

janblauer committed:
Update angular.yml

janblauer committed:
update to angular 18

janblauer commented:
@JoghurtCode ist behobe. bitte mergen :D

YoghurtBaum committed:
added api.yaml, userProfile Entity, UserProfileController

YoghurtBaum committed:
delete userprofilecontroller

YoghurtBaum committed:
Merge branch 'develop-dataTransfer' into develop-database

janblauer committed:
Merge pull request #33 from JoghurtCode/janblauer-patch-1

Update angular.yml

janblauer committed:
Merge pull request #35 from JoghurtCode/Update/Angular18

update to angular 18

janblauer committed:
Update angular.yml

Angular.yml in Github ändern

leonleon6871 committed:
calorie calculation class and method implemented

JoghurtCode committed:
Merge pull request #37 from JoghurtCode/creatingCalorieCalculatorClass

calorie calculation class and method implemented

janblauer committed:
Update angular.yml

Neuer versuchen frontend zu deployen

janblauer committed:
Neuer versuch zu deployen

janblauer committed:
Merge pull request #38 from JoghurtCode/Yaml-deployen

Neuer versuch zu deployen

janblauer committed:
Neuer versuch zu deployen- 2

janblauer committed:
Merge pull request #39 from JoghurtCode/Yaml-deployen

Neuer versuch zu deployen- 2

janblauer committed:
Neuer versuch zu deployen- 3

janblauer committed:
Wieder zurück gepatched

YoghurtBaum committed:
added entities, automated model building, config models in api.yaml

leonleon6871 committed:
Merge pull request #40 from JoghurtCode/develop-database

Develop database
